core/src/main/scala/za/co/absa/spline/harvester/logging/ObjectStructureDumper.scala

```diff
@@ -74,6 +76,7 @@ object ObjectStructureDumper {
     prevResult: DumpResult
   ): DumpResult = stack match {
     case Nil => prevResult
+    case head :: tail if head.depth > MaxDepth => objectToStringRec(extractFieldValue)(tail, visited, prevResult)
```

Comment:
In this case, should we output something like "... (the dump was cut here due to the max depth limit: 5)" ?

Reply:
I was thinking about dooing some more improvements in latter PR and just do quick fix to avoid the issue on databricks, but I improved it a little bit now.